### PR TITLE
Add Wikipedia Integration

### LINKED ISSUE #76: Wikipedia Integration
Add the Wikipedia tag from the osm data into the database and display a tooltip with the wiki link

### PR DESCRIPTION
**Pull request by pnancke**
closes #76

### CODE REVIEW

File: grails-app/assets/javascripts/trip-planner-helper.js [hunk 1]
```diff
@@ -107,18 +173,27 @@ function generateAutocompleteData(json) {
...
+
+            if (name == val.properties.street || name == val.properties.housenumber || name == val.properties.city
+                || name == val.properties.country) {
+                label = [address].join(', ');
+                currItem = {
+                    "label": label
+                };
+            } else {
+                label = [name, address].join(', ');
+                currItem = {
```
Comment (MZober1993) [inline grails-app/assets/javascripts/trip-planner-helper.js]: set currItem after if else branch

File: grails-app/assets/javascripts/trip-planner-helper.js [hunk 2]
```diff
@@ -29,49 +44,100 @@ function init() {
...
 
-function addMarker(lat, lon) {
-    var lonLat = new OpenLayers.LonLat(lon, lat)
-        .transform(new OpenLayers.Projection("EPSG:4326"), map.getProjectionObject());
-    markerLayer.addMarker(new OpenLayers.Marker(lonLat));
+function addMarker(lat, lon, label) {
+    var feature;
+    if (label) {
+        feature = new OpenLayers.Feature.Vector(
+            new OpenLayers.Geometry.Point(lon, lat).transform(epsg4326, projectTo),
```
Comment (MZober1993) [inline grails-app/assets/javascripts/trip-planner-helper.js]: can you extract this to a method with the parameter externalGraphic ?

File: grails-app/controllers/trip/planner/osm/api/Point.groovy [hunk 3]
```diff
@@ -13,6 +14,15 @@ class Point {
         this.lon = lon
     }
 
+    Point(Double lat, Double lon, String label) {
+        Objects.requireNonNull(lat)
+        Objects.requireNonNull(lon)
+
```
Comment (MZober1993) [inline grails-app/controllers/trip/planner/osm/api/Point.groovy]: is label an optional parameter? If not please check also with Objects.requireNonNull(label).

Comment (pnancke) [inline grails-app/controllers/trip/planner/osm/api/Point.groovy]: Label is optinal, not every POI has a wikipedia entry.

File: grails-app/domain/trip/planner/PointOfInterest.groovy [hunk 4]
```diff
@@ -144,6 +145,14 @@ class PointOfInterest {
         width nullable: true
     }
 
+    String getWikipedia() {
+        if (this.wikipedia != "NULL") {
+            this.wikipedia
+        }else {
```
Comment (MZober1993) [inline grails-app/domain/trip/planner/PointOfInterest.groovy]: I think you can remove the else branch of this statement.

File: grails-app/assets/javascripts/trip-planner-helper.js (hunk 2 above)
Comment (pnancke) [inline grails-app/assets/javascripts/trip-planner-helper.js]: Extracting one constructor call to a method?

**Pull request merged**